client/editor.lua

```diff
@@ -1,24 +1,38 @@
-RegisterCommand("record", function()
+-- We do this check as we don't want players using these commands due to crashing with escrowed maps.
+local shouldAllow, mapNames = true, {
+    'cfx-gabz-mapdata'
+}
+
+for i = 1, #mapNames do
+    local state = GetResourceState(mapNames[i])
+    if state ~= 'missing' and state ~= 'unknown' then
```

Comment:
Put this in a thread and check if it's started/starting as well to not make people have to move them out of the server or delete them to make this work

Reply:
The thread is to make sure it runs after all resources are started on the client, so to make sure it doesn't run before it starts, but after

Reply:
Updated to use started and starting checks instead, I didn't put them in a thread though as it's pointless and the commands would have to be placed inside it also